packages/dotnetmaui-icons/Microsoft.Maui.Icons.FluentUI/Microsoft.Maui.Icons.FluentUI.csproj

```diff
@@ -0,0 +1,167 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<ImplicitUsings>false</ImplicitUsings>
+		<Nullable>disable</Nullable>
+		<LangVersion>9.0</LangVersion>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+		<PackageId>Microsoft.Maui.Icons.FluentUI</PackageId>
+		<Title>Microsoft FluentUI System Icons for .NET MAUI</Title>
+		<Authors>Microsoft</Authors>
+		<Copyright>Copyright © Microsoft</Copyright>
+		<!--<PackageProjectUrl></PackageProjectUrl>-->
```

Comment:
if we know these now, we could add them :)

Reply:
fixed!